specs/interop/superchain-weth.md

```diff
@@ -0,0 +1,107 @@
+# Superchain WETH
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
```

Comment:
We will want a constants table that includes the addresses for the 2 new predeploys and an overview section similar to what you put together in the design doc, just framing the problem and the solution at a high level. We will want to mention the custom gas token chain compatibility